lib/functions/all_functions.dart

```diff
@@ -0,0 +1,54 @@
+import 'dart:convert';
+import 'package:http/http.dart' as http;
+
+import '../classes/user.dart';
+import '../classes/news.dart';
+import '../classes/top_stories.dart';
+import '../classes/comment.dart';
+
+Future<User> fetchUser(String user) async {
+  final response = await http.get(
+    Uri.parse(
+        'https://hacker-news.firebaseio.com/v0/user/$user.json?print=pretty'),
+  );
+  if (response.statusCode == 200) {
```

Comment:
Good idea to include error handling! It could be that `get` throws before this though, like when there is not internet connection. You might want to include it in the try block.